src/beans/Feedback.ts

```diff
@@ -1,10 +1,15 @@
 import axios from 'axios';
 import { BACKEND_BASE_URL } from '../constants';
 
-// Returns a axios chain
-const FormSubmit = ({ rating, feedback }) => {
+const FormSubmit = async ({
+  rating,
+  feedback
+}: {
+  rating: number;
+  feedback: string;
+}): Promise<void> => {
```

Comment:
Changing it to return a void Promise is a substantive change, but the only usage of this (in [src/components/Feedback/index.js](https://github.com/gt-scheduler/website/blob/development/src/components/Feedback/index.js#L20)) doesn't use the resolved value that axios gives, so it shouldn't change anything.